src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -0,0 +1,158 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project Sdk="Microsoft.DotNet.Arcade.Sdk">
+
+  <Import Project="./BuildTasks.props" />
+  <Import Project="../tools/SourceBuild/SourceBuildArcadeTools.targets"  Sdk="Microsoft.DotNet.Arcade.Sdk" />
+  <UsingTask TaskName="Microsoft.Linker.SourceBuild.Tarball_ReadSourceBuildIntermediateNupkgDependencies" AssemblyFile="$(SourceBuildTasksAssembly)" />
+  <UsingTask TaskName="Microsoft.Linker.SourceBuild.Tarball_WriteSourceRepoProperties" AssemblyFile="$(SourceBuildTasksAssembly)" />
+
+  <Target Name="CreateSourceTarball"
+    DependsOnTargets="
+      CreateTarballDir;
+      SetupSelfGithubInfo;
+      CloneRepoAndDependentsRecursive;
+      CopyTarballContent;
+      "/>
```

Comment:
Thoughts on emitting a message indicating the full tarball path upon success?

```
  <Target Name="CreateSourceTarball"
    DependsOnTargets="
      CreateTarballDir;
      SetupSelfGithubInfo;
      CloneRepoAndDependentsRecursive;
      CopyTarballContent;
      ">

    <Message Text="Tarball successfully created: $(TarballDir)" Importance="High" />
  </Target>
```

Reply:
Added.